ElementX/Sources/Screens/SecureBackup/SecureBackupRecoveryKeyScreen/View/SecureBackupRecoveryKeyScreen.swift

```diff
@@ -0,0 +1,221 @@
+    }
+    
+    @ViewBuilder
+    private var newRecoveryKeySection: some View {
+        generateRecoveryKeySection
+        
+        Spacer()
+        
+        if let recoveryKey = context.viewState.recoveryKey {
+            ShareLink(item: recoveryKey) {
+                Label(L10n.screenRecoveryKeySaveAction, icon: \.download)
+            }
+            .buttonStyle(.compound(.primary))
+            .simultaneousGesture(TapGesture().onEnded { _ in
+                context.send(viewAction: .keySaved)
+            })
+        }
+        
+        Button {
+            context.send(viewAction: .done)
+        } label: {
+            Text(L10n.actionDone)
+        }
+        .buttonStyle(.compound(.primary))
+        .disabled(context.viewState.recoveryKey == nil || context.viewState.doneButtonEnabled == false)
+    }
+    
+    private var generateRecoveryKeySection: some View {
+        VStack(alignment: .leading) {
+            Text(L10n.commonRecoveryKey)
+                .foregroundColor(.compound.textPrimary)
+                .font(.compound.bodySM)
+         
+            HStack {
+                if context.viewState.recoveryKey == nil {
+                    Button {
+                        context.send(viewAction: .generateKey)
+                    } label: {
+                        if context.viewState.mode == .setupRecovery {
+                            Text(L10n.screenRecoveryKeySetupGenerateKey)
+                                .foregroundColor(.compound.textPrimary)
+                                .font(.compound.bodyLGSemibold)
+                        } else {
+                            Text(L10n.screenRecoveryKeyChangeGenerateKey)
+                                .foregroundColor(.compound.textPrimary)
+                                .font(.compound.bodyLGSemibold)
+                        }
+                    }
+                } else {
+                    HStack(alignment: .top) {
+                        Text(context.viewState.recoveryKey ?? "")
+                            .foregroundColor(.compound.textPrimary)
+                            .font(.compound.bodyLG)
+                        
+                        Spacer()
+                        
+                        Button {
+                            context.send(viewAction: .copyKey)
+                        } label: {
+                            Image(asset: Asset.Images.copy)
+                                .foregroundColor(.compound.textSecondary)
+                        }
+                    }
+                }
+            }
+            .frame(maxWidth: .infinity)
+            .padding()
+            .background(Color.compound.bgSubtleSecondaryLevel0)
+            .clipShape(RoundedRectangle(cornerRadius: 8))
+            
+            HStack(alignment: .top) {
+                if context.viewState.recoveryKey == nil {
+                    CompoundIcon(\.info, size: .small, relativeTo: .body)
```

Comment:
To match the font its inline with below
```suggestion
                    CompoundIcon(\.info, size: .small, relativeTo: .bodySM)
```